hikey_debian.mk

```diff
@@ -281,12 +280,14 @@ grub-cleaner: grub-clean
 # Boot Image
 ################################################################################
 .PHONY: boot-img
-boot-img:
-ifeq ("$(wildcard $(BOOT_IMG))","")
-	echo "Downloading Debian HiKey boot image ..."
-	wget $(BOOT_IMG_URL) -O $(BOOT_IMG).gz
-	gunzip $(BOOT_IMG).gz
-endif
+boot-img: grub edk2
+	rm -f $(BOOT_IMG)
+	/sbin/mkfs.fat -F32 -n "boot" -C $(BOOT_IMG) 65536
+	mmd -i $(BOOT_IMG) EFI
+	mmd -i $(BOOT_IMG) EFI/BOOT
+	mcopy -i $(BOOT_IMG) $(EDK2_PATH)/Build/HiKey/$(EDK2_BUILD)_GCC49/AARCH64/AndroidFastbootApp.efi ::/EFI/BOOT/fastboot.efi
+	mcopy -i $(BOOT_IMG) $(OUT_PATH)/grubaa64.efi ::/EFI/BOOT/grubaa64.efi
+	chmod 644 $(BOOT_IMG)
```

Comment:
Based on https://git.linaro.org/ci/job/configs.git/tree/rpb-debian.yaml#n134.
I tried without it and it seems fine.
Will remove.